projects/canopy/src/lib/card/card-hero-img/card-hero-img.component.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  selector: 'lg-card-hero-img',
+  templateUrl: './card-hero-img.component.html',
+  styleUrls: [ './card-hero-img.component.scss' ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
+})
+export class LgCardHeroImageComponent {
+  @Input() coverImageUrl: string;
```

Comment:
I wonder if it might be slightly nicer to have a `cover` input as `boolean` defaulted to `false` and the other inputs as `src` and `alt`.

Reply:
Done. :)